### ISSUE TITLE
Regression: `Internal Error: Validation threw an exception` with assembly including linked templates.

### ISSUE THREAD

**Issue by edburns** [microsoft/vscode-azurearmtools#773]
[arm-oraclelinux-wls-admin-1.0.20-arm-assembly.zip](https://github.com/microsoft/vscode-azurearmtools/files/4754854/arm-oraclelinux-wls-admin-1.0.20-arm-assembly.zip)
Open the `mainTemplate.json` from the attachment in *vscode-azurearmtools* and you will see this error:

![message](https://user-images.githubusercontent.com/75821/84199584-6cb53f80-aa73-11ea-85dc-1b9a4841c85a.jpg)

Validated on 2020-06-09 17:00 EDT by @StephenWeatherford

**Comment by StephenWeatherford** [comment]
This repros with *any* linked templates if there's an attached parameter file. Does not repro in 0.9.x.

E.g.:
```
{
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
    "contentVersion": "1.0.0.0",
    "resources": [
        {
            "name": "aadLinkedTemplate",
            "type": "Microsoft.Resources/deployments",
            "apiVersion": "2019-10-01",
            "properties": {
                "mode": "Incremental",
                "templateLink": {
                    "uri": "https://foo"
                }
            }
        }
    ]
}
```
with this parameter file:
```
{
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {}
}
```

**Comment by StephenWeatherford** [comment]
Fixed by https://github.com/microsoft/vscode-azurearmtools/pull/806

**Comment by StephenWeatherford** [comment]
Fix released with 0.11.0
